packages/gatsby/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby",
   "description": "React.js Static Site Generator",
-  "version": "2.0.0-alpha.38",
+  "version": "1.9.261",
```

Comment:
This should still be 2.0.0.-alpha.38?

Reply:
You are defenitely right, thanks for noticing this in this enourmous diff :O